docs/extensions/hooks.md

```diff
@@ -191,8 +191,12 @@ export default ({ embed }, { env }) => {
 
 ### Filter Events
 
-| Name                           | Payload                              | Meta                                        |
+| Event                          | Payload                              | Meta                                        |
```

Comment:
This should stay as `Name` because the event is the thing itself that is triggered and the name of the event is what's specified in this column.

Reply:
Reason for me chancing name to Event since that is what it is called inside the output
```json
{
  "event": "event.name"
  "payload": "payload"
}
```

Reply:
Sorry, I can't follow. Where exactly would this output be returned?

Reply:
In the hooks

Reply:
I see where you're coming from but I think `Name` makes a bit more sense in the context of documentation as it's more descriptive which outweighs 1-to-1 matching to the event object.